### PR TITLE
fix(web): Fixed Navbar, Modal

### PR DESCRIPTION
**Pull request by mooyg-alt**
Fixed the Modal and Navbar component and changed the useOnClickOutside (just copied a snippet from use-hooks for now).

### CODE REVIEW

**Review by heybereket — APPROVED** [review]

**Review by f1shy-dev — CHANGES_REQUESTED** [review]
looks good, but do these first:
- fix constants.ts, youre meant to put the url in ENV
- remove the console.logs everywhere please

**Review by alexover1 — CHANGES_REQUESTED** [review]
You aren't supposed to change packages/web/src/lib/constants.ts

File: packages/web/src/components/common/Modal.tsx [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
 }
 
 export const Modal: React.FC<Props> = ({ open, closeHandler, children }) => {
   if (typeof window === 'undefined' || !open) return null;
 
-  const modalRef = useRef() as React.MutableRefObject<HTMLInputElement>;
+  const modalRef = useRef<HTMLDivElement | null>(null);
   useOnClickOutside(modalRef, () => closeHandler());
-
+  //fixed the modal
```
Comment (BronzW) [inline packages/web/src/components/common/Modal.tsx:15]: Why did you add this comment?

**Review by BronzW — APPROVED** [review]
